lib/internal/Magento/Framework/DB/Ddl/Table.php

```diff
@@ -78,6 +78,20 @@ class Table
 
     const ACTION_SET_DEFAULT = 'SET DEFAULT';
 
+    /**#@+
+     * Option keys for column options
```

Comment:
Such syntax is obsoleted, see http://devdocs.magento.com/guides/v2.2/coding-standards/docblock-standard-general.html#DocBlock-templates

Reply:
Fixed